### ISSUE TITLE
Sync issue

### ISSUE THREAD

**Issue by dotevo** [redmatrix/hubzilla#557]
Hi,

I'd found for example that post on Hubzilla (one day old). No comments/likes here:
![1](https://cloud.githubusercontent.com/assets/246080/19511270/eb4637f0-95e6-11e6-9de6-081dd0bb57c0.png)

But when I clicked into it I found many comments:
![2](https://cloud.githubusercontent.com/assets/246080/19511276/ee69b1d2-95e6-11e6-9c83-f544c440ef51.png)

It looks like synchronization issue in federation servers.

BR,
Adam

**Comment by git-marijus** [comment]
Is the hubzilla post from the discovery tab or from your stream?

**Comment by dotevo** [comment]
It is from "Commented Order" but that post is from Diaspora Protocol Settings -> Followed hashtags

**Comment by git-marijus** [comment]
Ah... That is probably just beeing imported...

**Comment by zotlabs** [comment]
It's a known issue with Diaspora Relay Servers, which are still somewhat experimental and not yet universally supported within Diaspora. We have followed the relay protocol precisely and have subscribed to tags successfully, but the tagged content is only intermittently delivered to hubzilla by the Diaspora relay servers. If they don't send us the comments or the tagged posts we have subscribed to, there isn't a lot we can do about it.
